lib/internal/Magento/Framework/App/Config.php

```diff
@@ -31,6 +36,8 @@ class Config implements ScopeConfigInterface
      */
     private $types;
 
+    public $isSingleStoreModeEnabled;
```

Comment:
1. That should not be public property (change to private)
2. Add PHPDoc for the property